examples/mvn/single-module/pom.xml

```diff
@@ -13,25 +13,34 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <java.version>1.8</java.version>
-        <scala.version>2.12.6</scala.version>
+        <scala.version>2.12.8</scala.version>
+        <scala.binary.version>2.12</scala.binary.version>
+        <scapegoat.version>1.3.8</scapegoat.version>
 
         <sonar.projectName>[example] Maven Single Module</sonar.projectName>
         <sonar.projectKey>example-mvn-single-module</sonar.projectKey>
         <sonar.sources>src/main/scala</sonar.sources>
         <sonar.tests>src/test/scala</sonar.tests>
         <sonar.junit.reportPaths>target/surefire-reports</sonar.junit.reportPaths>
         <sonar.sourceEncoding>UTF-8</sonar.sourceEncoding>
-        <sonar.scala.version>2.12</sonar.scala.version>
-        <sonar.scala.scoverage.reportPath>target/scoverage.xml</sonar.scala.scoverage.reportPath>
+        <sonar.scala.version>${scala.binary.version}</sonar.scala.version>
+        <sonar.scala.scoverage.reportPath>${project.build.directory}/scoverage.xml</sonar.scala.scoverage.reportPath>
+        <sonar.scala.scapegoat.reportPath>${project.build.directory}/scapegoat.xml</sonar.scala.scapegoat.reportPath>
```

Comment:
Apparently using the `${project.build.directory}` property is a best practice in maven. Thus, I used it everywhere.

Maybe, we should add a note about this in the README too.
It is mandatory for multi-module projects at least. If not used, the `scapegoat.xml` report will be generated only for the last compiled module, and will be placed in the root target folder, so it will be not picked by sonar,